CharacterCreation/03_Feats.txt

```diff
@@ -675,26 +675,30 @@ have no prerequisites.
 
 0:	+10m to range R2. If this would cause R2 to be greater than R3, 
 	also increase R3 to match the new R2.
-12:	May use Weapons - Assault/Battle Rifle in place of Spotting when looking through your 
+12:	May use Weapons - Assault/Battle Rifle (PER) in place of Spotting when looking through your 
 	rifle's scope (warning, takes longer than wider field-of-view optics).
 24:	Halve the jam chance (applies after additive modifiers, round down, may be zero) of your 
 	assault/battle rifle for the first 3 rounds of its use per combat, once per short rest 
 	(i.e. maintenance & cleaning oportunity).
 36:	+1 Accuracy while firing in burst mode and +2 Accuracy while firing in automatic mode 
 	while benefiting from any kind of cover against your target.
 48:	+5m to range R1, +10m to range R3.
-60:	+10% Base damage; You may critically hit as normal while firing in burst mode, and may 
+60:	+10% Base damage. You may critically hit as normal while firing in burst mode, and may 
 	critically hit with half your normal critical range while firing in automatic mode.
 
 == Long Rifle Expertise ==
 
-0:	-5m to minimum range, +10% meters to R2 & R3		
-12:	-10% reload miss chance			
-24:	+10% to perception checks to spot enemies beyond your long rifle's R2 range			
-36:	-0.5 miss chance @ all ranges			
-48:	Gain feat "Beware the Wrath of a Patient Man": shots fired as a held reaction have a -1 miss chance.			
-60:	+10% Base damage; Once per turn may change aim between targets within 30 degrees 
-	as a free action by passing a DC 75 dexterity check
+0:	+10% to range R3.
+12:	May use Weapons - Long Rifle (STR) in place of Athletics in order to get to hard-to-reach
+	places, like towers or treetops, that can serve as sniper nests.
+24:	While prone, benefiting from at least light cover, and able to see at least one enemy, 
+	gain an additional +1 cover bonus.
+36:	Gain a stacking +0.5 Accuracy each time you consecutively spend an action shooting a 
+	single target without losing line of sight to it.
+48:	Halve any minimum range associated with your long rifle and attachments, to a max reduction 

```

Comment:
I bemoan a little bit the removal of "wrath of a patient man", which I wrote, but I like the pattern that you're keeping.

Reply:
Honestly, it could work as its own standalone feat. As far as I'm concerned, feel free to reintroduce it as such!